lib/ansible/modules/cloud/vmware/vmware_guest_disk.py

```diff
@@ -151,6 +187,50 @@
   delegate_to: localhost
   register: disk_facts
 
+- name: Add disks with specified shares to the virtual machine
+  vmware_guest_disk:
+    hostname: "{{ vcenter_hostname }}"
+    username: "{{ vcenter_username }}"
+    password: "{{ vcenter_password }}"
+    datacenter: "{{ datacenter_name }}"
+    validate_certs: no
+    disk:
+      - size_gb: 10
```

Comment:
could you please just create a 1GB volume, this is enough to validate the module. And this will reduce the constraint on the test environment.

Reply:
@goneri
as part of integration test we cant maintain 1gb of size in all test cases, as while updating the disk attributes, size should be more than the existing disk size , for now i am giving single digit disk sizes like 1,2,3,4